docs/changelog/3530.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Prevent tox from hanging upon exit due to orphaned build threads and
```

Comment:
Align to 120 charachters.

Reply:
Alright, although other newsfragments in `main` are not aligned to 120 characters.